### PR TITLE
Ambiguous Documentation fix for guvectorize.

### PR DESCRIPTION
**Pull request by rteja1113**
The wording is ambiguous in the second bullet point below the example code in the documentation for `guvectorize`. This fix resolves that by explicitly saying that the example for `guvectorize` only demonstrates int64 arrays but it can support every signature as in `vectorize`

### CODE REVIEW

File: docs/source/user/vectorize.rst [hunk 1]
```diff
@@ -164,8 +164,8 @@ There are two things there:
   array, a scalar (symbolically denoted by the empty tuple ``()``) and
   returns a *n*-element one-dimension array;
 
-* the list of supported concrete *signatures* as in ``@vectorize``; here we
-  only support ``int64`` arrays.
+* the list of supported concrete *signatures* are the same as in ``@vectorize``; here
```
Comment (stuartarchibald) [inline docs/source/user/vectorize.rst]:
```suggestion
* the list of supported concrete *signatures* as per ``@vectorize``; here,
```

I think this statement needs to allude to that the provision of concrete signatures is the same as noted for `@vectorize`.

File: docs/source/user/vectorize.rst [hunk 2]
```diff
@@ -164,8 +164,8 @@ There are two things there:
...
+  in the above example we demonstrate ``int64`` arrays.
```
Comment (stuartarchibald) [inline docs/source/user/vectorize.rst]:
```suggestion
  as in the above example, we demonstrate ``int64`` arrays.
```

**Comment by rteja1113** [comment]
> Thanks for the patch, I've made a couple of suggestions.

Thanks for the suggestions @stuartarchibald .

**Comment by stuartarchibald** [comment]
@esc please could you check this as it now also contains my suggestions. Thanks.

**Comment by esc** [comment]
@stuartarchibald I am happy with the wording. CI seems to be failing because of anaconda.org timeouts, will re-trigger now.

**Comment by esc** [comment]
Can be marked for merge as soon as CI goes green.

**Pull request merged**